### DISCUSSION TITLE
Getting more features to sveltekit that the lexical playground already offers

### DISCUSSION THREAD

**Discussion by Rar9** [umaranis/svelte-lexical#61] [Q&A]
Is it possible to get more features to this svelte version that the lexical playground already offers?

**Comment by umaranis** [comment]
Hi @Rar9 ,

I will be working on porting more features. Any features you want to be addressed on priority?

**Reply by Rar9** [reply]
Tables & columns, image & link handeling, hr & page break, basically the basics like color. 

Example for html & json data import /export handeling
